index.js

```diff
@@ -27,12 +27,12 @@ var getConfig = function() {
 };
 
 function normalizePath(full) {
-  var dir_name = full;
-
-  if (dir_name[0] !== '/') {
-    dir_name = path.normalize(currentCWD + '/' + dir_name);
+  var dir_name = path.resolve(full);
+  
+  if(dir_name.startsWith('\\\\?\\')){
+      dir_name = dir_name.substring(4, dir_name.length);
   }
-  dir_name += (dir_name[dir_name.length - 1] === '/') ? '' : '/';
+  dir_name += (dir_name[dir_name.length - 1] === path.sep) ? '' : path.sep;
   return dir_name;
 }
```

Comment:
A few things here:

* Please use the same spacing as in the rest of the file.
* `startsWith` isn't valid in older Node versions, if we are to use this then a major version bump is required.
* Build is failing due to lack of coverage since no test was added for this feature.